src/utils/utils.cpp

```diff
@@ -189,10 +189,11 @@ bool load_rawfile(const std::string& rawfilename, const Eigen::RowVector3i& dims
     return true;
 }
 
-void init_opengl_debug() {
+// void init_opengl_debug(void (*)(GLenum, GLenum, GLuint, GLenum, GLsizei, const GLchar *, const void *) callback) {
```

Comment:
rm commented code

Reply:
Sorry about that. Should have known better